README.md

```diff
@@ -1 +1,138 @@
+
+4. Create `.env` file:
+
+```
+OPEN_RUNTIMES_PROXY_ALGORITHM=round-robin
+OPEN_RUNTIMES_PROXY_EXECUTORS=whoami1,whoami2
+OPEN_RUNTIMES_PROXY_HEALTHCHECK=disabled
+OPEN_RUNTIMES_PROXY_SECRET=proxy-secret-key
+OPEN_RUNTIMES_PROXY_HEALTHCHECK_INTERVAL=5000
+OPEN_RUNTIMES_PROXY_ENV=development
+OPEN_RUNTIMES_PROXY_EXECUTOR_SECRET=executor-secret-key
+OPEN_RUNTIMES_PROXY_LOGGING_PROVIDER=
+OPEN_RUNTIMES_PROXY_LOGGING_CONFIG=
+```
+
+> Notice we disabled health check. We recommend keeping it `enabled` and implementing proper health check endpoint
+
+5. Start Docker containers:
+
+```bash
+docker compose up -d
+```
+
+6. Send a HTTP request to proxy server:
+
+```bash
+curl -H "Authorization: Bearer proxy-secret-key" -X GET http://localhost:9800/
+```
+
+Run the command multiple times to see request being proxied between both whoami servers. You can see `Hostname` changing the value.
+
+> Noitce we provided authorization header as configured in `.env` in `OPEN_RUNTIMES_PROXY_SECRET`.
+
+7. Stop Docker containers:
+
+```bash
+docker compose down
+```
+
+## Environment variables
+
+| Variable name                           | Description                                                                                                                               |
+|-----------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------|
+| OPEN_RUNTIMES_PROXY_ALGORITHM           | Proxying algorithm. Supports `round-robin`, `random`.                                                                                     |
+| OPEN_RUNTIMES_PROXY_EXECUTORS           | Comma-separated hostnames of servers under the proxy.                                                                                     |
+| OPEN_RUNTIMES_PROXY_HEALTHCHECK | Health check by HTTP request to /v1/health. 'enabled' by default. To disable, set to 'disabled'.                                          |
+| OPEN_RUNTIMES_PROXY_HEALTHCHECK_INTERVAL       | Delay in milliseconds between health checks. 10000 by default. Only relevant if OPEN_RUNTIMES_PROXY_HEALTHCHECK is 'enabled'.     |
+| OPEN_RUNTIMES_PROXY_ENV                 | Runtime environment. 'production' or 'development'. Development may expose debug information and is not recommended on production server. |
+| OPEN_RUNTIMES_PROXY_SECRET              | Secret that needs to be provided in `Authroization` header when talking to proxy.                                                         |
+| OPEN_RUNTIMES_PROXY_EXECUTOR_SECRET     | String provided as `Authorization` header by proxy when sending request to executor.                                                      |
+| OPEN_RUNTIMES_PROXY_LOGGING_PROVIDER    | Logging provider. Supports `sentry`, `appsignal`, `raygun`, `logown`. Leave empty for no cloud logging.                                   |
```

Comment:
```suggestion
| OPEN_RUNTIMES_PROXY_LOGGING_PROVIDER    | Logging provider. Supports `sentry`, `appsignal`, `raygun`, `logowl`. Leave empty to disable errro logging.                                   |
```

Reply:
It does not fully disable logging. You still get logging into console. Logs are just not reported to cloud providers.